### ISSUE TITLE
Various grafana-bridge Dockerfile updates

### ISSUE THREAD

**Issue by maroche15** [IBM/ibm-spectrum-scale-bridge-for-grafana#43]
- Lock base UBI8.4 image to `ubi8.4-206`
- Produce requirements.txt with versioned packages that can be used for python package installation during image build.
- Copy apache2 license into grafana-bridge container

**Comment by Helene** [comment]
@maroche15 I'm not sure, I understand the requirement "Lock base UBI8.4 image to ubi8.4-206" correctly. You mean the Dockerfile from image should look like below?
```
FROM registry.access.redhat.com/ubi8.4-206/ubi
```

**Comment by maroche15** [comment]
The image would look like:
```
registry.access.redhat.com/ubi8/ubi:8.4-206
```

**Comment by Helene** [comment]
@maroche15 the requested changes have been included in the release version [v7.0.3](https://github.com/IBM/ibm-spectrum-scale-bridge-for-grafana/releases/tag/v7.0.3)
